fa-products-service-sand-ne-001/http-get-product-by-id/index.ts

```diff
@@ -0,0 +1,15 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions";
+import productList from "../constant";
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+    context.log('HTTP trigger function http-get-product-by-id processed a request.');
+    const { productId } = req.params;
+    const product = productList?.find(item => item.id === productId);
+
+    context.res = {
+        status: product? 200 : 404, /* Defaults to 200 */
+        body: product ? product : console.error("Product not found")        
```

Comment:
In Azure Functions context should be used instead of the console
<img width="773" alt="image" src="https://github.com/Renatalina/azure-fullstack/assets/1252687/b6000562-ced8-48df-81ef-38eb1e165558">
https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=javascript%2Cwindows%2Cazure-cli&pivots=nodejs-model-v4#logging
https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=javascript%2Cwindows%2Cazure-cli&pivots=nodejs-model-v4#log-levels